modules/page-memory/src/main/java/org/apache/ignite/internal/pagememory/persistence/WriteSpeedFormatter.java

```diff
@@ -48,14 +48,25 @@ private WriteSpeedFormatter() {
         // no-op
     }
 
+    /**
+     * Format write speed in MB/sec.
+     *
+     * @param bytes Total number of bytes.
+     * @param durationInSeconds Duration in seconds.
+     * @return Formatted write speed.
+     */
+    public static String formatWriteSpeed(long bytes, long durationInSeconds) {
+        return formatWriteSpeed((double) bytes / Math.max(1, durationInSeconds));
```

Comment:
`Math.max(1, durationInSeconds)` - what does it mean?

Reply:
This way I eliminate division by zero.

Reply:
Fix it.